UIKit_HM/UIKit_HM/LoginViewController.swift

```diff
@@ -0,0 +1,74 @@
+import UIKit
+
+final class LoginViewController: UIViewController {
+	
+	// MARK: - Visual Components
+	
+	private let numberLabel = UILabel()
+	private let numberTextField = UITextField()
+	private let passwordLabel = UILabel()
+	private let passwordTextField = UITextField()
+	private let enterButton = UIButton()
+	
+	// MARK: - Life cycle
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		setUpView()
+	}
+	
+	// MARK: - Private methods
+	
+	private func setUpView() {
+		var titleImageView = UIImageView()
+		let image = UIImage(systemName: "cloud.fill")
+		titleImageView = UIImageView(image: image)
```

Comment:
мне кажетлся лишние действие, ты создаешь два раза имедж вью
```suggestion
		let image = UIImage(systemName: "cloud.fill")
		let titleImageView = UIImageView(image: image)
```